src/data/thermal-conductivity-constants.ts

```diff
@@ -2050,6 +2050,730 @@ export const DIPPRThermalConductivityDictionary: Record<
+  },
+
+  HydrogenChloride: {
+    name: 'Hydrogen chloride',
+    formula: 'ClH',
+    CAS: '7647-01-0',
+    molecularWeight: 36.46094,
+    C1: 0.8045,
+    C2: -0.002102,
+    minimumTemperature: 273.15,
+    maximumTemperature: 323.15,
+    thermalConductivityAtMinimumTemperature: 0.2303,
+    thermalConductivityAtMaximumTemperature: 0.1252,
+  },
+
+  HydrogenCyanide: {
+    name: 'Hydrogen cyanide',
+    formula: 'CHN',
+    CAS: '74-90-8',
+    molecularWeight: 27.02534,
+    C1: 0.43454,
+    C2: -0.0007008,
+    minimumTemperature: 259.83,
+    maximumTemperature: 298.85,
+    thermalConductivityAtMinimumTemperature: 0.2525,
+    thermalConductivityAtMaximumTemperature: 0.2251,
+  },
+
+  HydrogenFluoride: {
+    name: 'Hydrogen fluoride',
+    formula: 'FH',
+    CAS: '7664-39-3',
+    molecularWeight: 20.0063432,
+    C1: 0.7516,
+    C2: -0.0010874,
+    minimumTemperature: 189.79,
+    maximumTemperature: 394.45,
+    thermalConductivityAtMinimumTemperature: 0.5452,
+    thermalConductivityAtMaximumTemperature: 0.3227,
+  },
+
+  HydrogenSulfide: {
+    name: 'Hydrogen sulfide',
+    formula: 'H2S',
+    CAS: '04-06-7783',
+    molecularWeight: 34.08088,
+    C1: 0.4842,
+    C2: -0.001184,
+    minimumTemperature: 193.15,
+    maximumTemperature: 292.42,
+    thermalConductivityAtMinimumTemperature: 0.2555,
+    thermalConductivityAtMaximumTemperature: 0.138,
+  },
+
+  IsobutyricAcid: {
+    name: 'Isobutyric acid',
+    formula: 'C4H8O2',
+    CAS: '79-31-2',
+    molecularWeight: 88.10512,
+    C1: 0.21668,
+    C2: -0.0002556,
+    minimumTemperature: 227.15,
+    maximumTemperature: 482.75,
+    thermalConductivityAtMinimumTemperature: 0.1586,
+    thermalConductivityAtMaximumTemperature: 0.0933,
+  },
+
+  IsopropylAmine: {
+    name: 'Isopropyl amine',
+    formula: 'C3H9N',
+    CAS: '75-31-0',
+    molecularWeight: 59.11026,
+    C1: 0.237,
+    C2: -0.000332,
+    minimumTemperature: 177.95,
+    maximumTemperature: 305.55,
+    thermalConductivityAtMinimumTemperature: 0.1779,
+    thermalConductivityAtMaximumTemperature: 0.1356,
+  },
+
+  MalonicAcid: {
+    name: 'Malonic acid',
+    formula: 'C3H4O4',
+    CAS: '141-82-2',
+    molecularWeight: 104.06146,
+    C1: 0.28231,
+    C2: -0.00024019,
+    minimumTemperature: 409.15,
+    maximumTemperature: 580.0,
+    thermalConductivityAtMinimumTemperature: 0.184,
+    thermalConductivityAtMaximumTemperature: 0.143,
+  },
+
+  MethacrylicAcid: {
+    name: 'Methacrylic acid',
+    formula: 'C4H6O2',
+    CAS: '79-41-4',
+    molecularWeight: 86.08924,
+    C1: 0.2306,
+    C2: -0.00025201,
+    minimumTemperature: 288.15,
+    maximumTemperature: 530.0,
+    thermalConductivityAtMinimumTemperature: 0.158,
+    thermalConductivityAtMaximumTemperature: 0.097,
+  },
+
+  Methane: {
+    name: 'Methane',
+    formula: 'CH4',
+    CAS: '74-82-8',
+    molecularWeight: 16.0425,
+    C1: 0.41768,
+    C2: -0.0024528,
+    C3: 3.56e-6,
```

Comment:
```suggestion
    C3: 3.5588e-6,
```

Don't loose precision in data.